<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioControllerWidget</class>
 <widget class="QWidget" name="AudioControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(80, 80, 80, 0.25), stop:1 rgba(130, 130, 130, 0.25));</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="styleSheet">
      <string notr="true">#mainFrame {
	border-radius: 5px;
	border: 2px solid white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="volumeLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="volume">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Plain</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="muteButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">#muteButton {
	border-radius: 5px;
	border: 2px solid white;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::AudioVolumeMuted"/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="durationLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>50</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="durationSlider">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="durationLabel">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#durationLabel {
	background: transparent;
}</string>
            </property>
            <property name="text">
             <string>---</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mediaButtonLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="skipBackButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#skipBackButton {
	border-radius: 10px;
	border: 2px solid white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPauseButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#playPauseButton {
	border-radius: 10px;
	border: 2px solid white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="skipButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#skipButton {
	border-radius: 10px;
	border: 2px solid white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadNewAudioFileButton">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">#loadNewAudioFileButton {
	border-radius: 5px;
	background-color: rgb(0, 170, 255);
}</string>
          </property>
          <property name="text">
           <string>Load New Audio File</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
